README.md

```diff
@@ -34,7 +34,8 @@ If you use this software in an academic work, please cite:
 
 ## Installation notes
 
-DBoW2 requires [DLib](https://github.com/dorian3d/DLib), which is automatically installed if it cannot be found in the system. You can also find it in [my repository](https://github.com/dorian3d/DLib).
+ 
```

Comment:
Hi, tomorrow i Will keep working on the lib. Ill check Windows compatibility and few other things. I am also working with dloopdetector lib, wich i have removed dlib dependencies, and i would strongly suggest to merge with dbow2, since it is only adding one header file.
Ill come back to you tomorrow.
El 17 ago. 2016 4:26 p. m., Pierre Moulon <notifications@github.com> escribió:In README.md:
> @@ -34,7 +34,8 @@ If you use this software in an academic work, please cite:
>  
>  ## Installation notes
>  
> -DBoW2 requires [DLib](https://github.com/dorian3d/DLib), which is automatically installed if it cannot be found in the system. You can also find it in [my repository](https://github.com/dorian3d/DLib).
> + 

You must also remove the ${DLib_LIBS}

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.